app.js

```diff
@@ -171,6 +230,7 @@ function buildAPrettyChart(){
   });
 }
 
+window.addEventListener('load',onLoadValues);
```

Comment:
Nitpick: I'd probably call this `loadValues` rather than `onLoadValues`. Functions should almost always start with verbs.